packages/firestore/src/platform_node/grpc_connection.ts

```diff
@@ -82,11 +75,6 @@ function createMetadata(
 // tslint:disable-next-line:no-any
 type GeneratedGrpcStub = any;
```

Comment:
I am wondering if it makes sense to define the type for GeneratedGrpcStub. It should end up looking something like
https://github.com/googleapis/nodejs-firestore/blob/master/dev/test/util/helpers.ts#L65 and might prevent issues like this in the feature.

Reply:
Oh, nice...  That is interesting, but is probably hard to do in a useful way right now since we rely on having a relatively-untyped `Connection` interface that's shared by WebChannel and gRPC (e.g. https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore/src/remote/connection.ts#L46)...  But I'll keep that in the back of my head in case we ever feel like reworking that.